flixel/system/FlxSound.hx

```diff
@@ -713,6 +726,18 @@ class FlxSound extends FlxBasic
 		}
 		return _time = time;
 	}
+	
+	private function set_loopTime(value:Float):Float
+	{
+		loopTime = FlxMath.bound(value, 0, length);
+		set_endTime(endTime);
+		return loopTime;
+	}
+	
+	private function set_endTime(value:Float):Float
+	{
+		return endTime = (value <= loopTime) ? length : value;
```

Comment:
I don't think this check is still correct - `value <= loopTime` only matters if the sound is `looped`. This probably means that we'd need a `set_looped()` as well.